ci/test.sh

```diff
@@ -41,6 +41,19 @@ mvn -B verify --file pom.xml \
 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
 
 echoinfo "Test complete"
+
+echoinfo "Uploading test results"
+
+export CODACY_PROJECT_TOKEN="$(cat /workspace/ci/codacy-project-token | sed -e 's/\n//g')"
+for f in $(find . -type f -name 'jacoco.xml'); do
+    bash <(curl -Ls https://coverage.codacy.com/get.sh) report -l Java -r $f --partial
```

Comment:
Would it make sense to store `curl -Ls https://coverage.codacy.com/get.sh` in a file since we're running it in a loop?

Reply:
It actually uses a cache
https://github.com/codacy/codacy-coverage-reporter/blob/master/get.sh#L73

Reply:
yes, but we're still hitting HTTP multiple times for the `get.sh` script... unnecessary traffic :shrug:

Reply:
True. Moved.